java/statestore/src/test/java/sleeper/statestore/dynamodb/DynamoDBStateStoreIT.java

```diff
@@ -738,7 +738,7 @@ public void shouldAtomicallyUpdateStatusToReadyForGCAndCreateNewActiveFile() thr
         }
 
         @Test
-        public void shouldAtomicallyUpdateStatusToReadyForGCAndCreateNewActiveFilesForSplittingJob() throws StateStoreException {
+        public void shouldAtomicallyUpdateStatusToReadyForGCAndCreateNewActiveFilesForFileSplitting() throws StateStoreException {
```

Comment:
I'm not sure this is valid any more, as atomicallyUpdateFilesToReadyForGCAndCreateNewActiveFiles isn't used for splitting file references now.

Reply:
I'll remove this test, i've raised issue #1719 to fully replace this method signature on the state store with one that takes a single output file